src/src.pro

```diff
@@ -18,8 +18,8 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5) {
-  error("requires Qt 5.0 or later")
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
It's an undocumented option in qmake4 I think. Either way, has the desired effect.

Major version is _less than_ 6, so 5 is OK. 4 is also OK... hmm, damnit, 4.8 will be OK according to this. qmake kind of sucks in this regard, I think I'll have to use 2 lines for this.